src/modules/convert-array-of-objects-to-csv.js

```diff
@@ -15,7 +15,7 @@ export const convertArrayOfObjectsToCSV = (data, { header, separator }) => {
 
   array.forEach((row, idx) => {
     const thisRow = Object.keys(row);
-    if (!header && idx === 0) {
+    if (header === undefined && idx === 0) {
```

Comment:
Hello,

thanks for your contribution 👍 

I have to dig into the project again. But just for clarification, doesnt `!header` does the same as `header === undefined` but include things like `0`, `null`, `etc...`

![Screenshot 2020-11-27 at 17 15 34](https://user-images.githubusercontent.com/14862422/100468251-55771680-30d4-11eb-9fff-5dfe7bf7435a.png)

Reply:
The intent is to allow Null or False (and explicit negative) to signify not to include the header, but when the value is not specified at all... to automatically include the default

Reply:
Ah okay, so if its null or undefined it should skip this line and not go into the if stmt.

Reply:
By changing the logic to identify an explicit false we're able to have the following:

```
const dataArrays = [ {a: 1, b: 2}, {a: 3, b: 4} ];
const csvFromArrayOfArrays = convertArrayToCSV(dataArrays, { header: false });
```

With output of:
```
1,2
3,4
```

Versus the normal behavior of:

```
const dataArrays = [ {a: 1, b: 2}, {a: 3, b: 4} ];
const csvFromArrayOfArrays = convertArrayToCSV(dataArrays, { header: false });
```

With output of:
```
a,b
1,2
3,4
```

Reply:
I see, that looks legit. I have to take a look myself, but overall I am looking forward to getting this PR merged.